### PR TITLE
Add webpack-visualizer-plugin to dev_mode webpack config

### PR DESCRIPTION
**Pull request by gnestor**
This makes it easy for jupyterlab developers to analyze the dev_mode app bundle.

This adds an "analyze" script so you can run `jlpm analyze` or `jlpm analyze:prod` from the project root and it will build the dev_mode bundle and open the results in the browser. 

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,9 @@
   "scripts": {
     "pre-publish": "jlpm && cd buildutils && npm run build && cd .. && node buildutils/lib/prepublish.js",
     "add:sibling": "node buildutils/lib/add-sibling.js",
+    "analyze": "jlpm run analyze:dev",
+    "analyze:dev": "cd dev_mode && jlpm run build && open static/stats.html",
+    "analyze:prod": "cd dev_mode && jlpm run build:prod && open static/stats.html",
```
Comment (vidartf) [inline package.json]: Is there a platform independent version of `open`?

Comment (gnestor) [reply package.json]: It doesn't appear like there is: https://dwheeler.com/essays/open-files-urls.html

We could use an npm package like [opn](https://www.npmjs.com/package/opn)

**Comment by gnestor** [comment]
Some notes:

- xterm could be lazy loaded
- santitize-html too big
    - [ ]  What's it doing with postcss?
- moment is too big
    - [ ]  Import submodule
- vim keymap for codemirror
    - [ ]  Lazy load keymaps

**Comment by gnestor** [comment]
Some notes:

- xterm could be lazy loaded
- santitize-html too big
    - [ ]  What's it doing with postcss?
- moment is too big
    - [ ]  Import submodule
- vim keymap for codemirror
    - [ ]  Lazy load keymaps

**Comment by gnestor** [comment]
This adds cross-platform support for "open" and only creates a `stats.html` when the analyze script is run (vs. ust `jlpm build:dev`).

**Comment by ian-r-rose** [comment]
Thanks @gnestor!

**Pull request merged**